app/_hub/kong-inc/ai-proxy/how-to/_llama2.md

```diff
@@ -108,7 +108,7 @@ plugins:
         provider: "llama2"
         name: "llama2"
         llama2_format: "ollama"
-        upstream_url: "http://llama2-server.local:11434/v1/chat"
+        upstream_url: "http://llama2-server.local:11434/api/chat"
```

Comment:
Looks like the config on line 92 needs updating as well (same thing, but in cURL format).